.bazelversion

```diff
@@ -1 +1 @@
-6.1.1
+7.0.0-pre.20230724.1
```

Comment:
Note: I don't think we actually need to be using the latest rolling release here.